src/components/DownloadStudySeriesDialog.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { useCallback, useState } from 'react'
+import { PaperClipOutlined } from '@ant-design/icons'
+
+const getConfig = (): {
+  description: string
+  instructions: Array<{
+    command: string
+    label: string
+  }>
+} => {
+  const defaultConfig = {
```

Comment:
Can we expose this to slim's configuration? so then we can customize these values without having to build the viewer again